### PR TITLE
[TOML] add Stringify feature

### PR DESCRIPTION
**Pull request by zekth**
Added the dump feature for the toml. Corrected an optionnal parameter for `string pad` too.

### CODE REVIEW

File: toml/README.md [hunk 1]
```diff
@@ -93,3 +95,17 @@ const tomlObject = parseFile("file.toml");
...
+```ts
+import { dump } from "./parser.ts";
+const obj = {
+  bin: [
+    { name: "deno", path: "cli/main.rs" },
+    { name: "deno_core", path: "src/foo.rs" }
+  ],
+  nib: [{ name: "node", path: "not_found" }]
+};
+const tomlString = dump(obj);
```
Comment (ry) [inline toml/README.md]: Rename to “stringing” ?

Comment (zekth) [reply toml/README.md]: Most of the time the name `dump` is used. For example:
- https://www.npmjs.com/package/js-yaml
- https://pyyaml.org/wiki/PyYAMLDocumentation
- https://pypi.org/project/toml/

Comment (zekth) [reply toml/README.md]: But i can rename if you want.

Comment (ry) [reply toml/README.md]: Yes please rename.

**Pull request merged**